models/intermediate/int_amazon_ads__portfolio_history.sql

```diff
@@ -15,11 +15,13 @@ with portfolios as (
         {% for column in columns %}
             {# set null for each column #}
             {%- if column['name'] == 'id' -%}
-                cast(null as {{ dbt.type_bigint() }}) as portfolio_id
+                cast(null as {{ dbt.type_string() }}) as portfolio_id
```

Comment:
For this int model, it is used by other models (like [here](https://github.com/fivetran/dbt_amazon_ads/blob/d9b18cf4d77c517dcfcebee3a66b7b917d7c1186/models/amazon_ads__campaign_report.sql#L14).) I recall we did this so we didn't have to add logic to each model that brought in portfolios, even if it was turned off (like for the data we have.)

Reply:
Ahhh yes I was only looking at the portfolio model and completely missed looking at the campaign report. Yeah I think it would be worthwhile to create a FR to clarify this or leverage in more places if needed. No need to incorporate in this update.

Reply:
Added the issue!